src/components/HathorFlatList.component.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {
+  FlatList, FlatListProps, StyleSheet, View,
+} from 'react-native';
+import { COLORS } from '../styles/themes';
+
+/**
+ * param {FlatListProps} props
+ */
+export const HathorFlatList = (props) => (
+  <View style={styles.wrapper}>
+    <FlatList
+      ItemSeparatorComponent={ItemSeparator}
+      // Introduced last to allow overwrite
+      {...props}
+    />
+  </View>
+);
+
+const ItemSeparator = () => (
+  <View style={{ width: '100%', height: 1, backgroundColor: COLORS.borderColor }} />
```

Comment:
Move those styles to the stylesheet you created below

Reply:
It was moved.